pkgs/build-support/dotnet/make-nuget-deps/default.nix

```diff
@@ -41,6 +41,22 @@ lib.makeOverridable(
           unpackPhase = ''
             unzip -nq $src
             chmod -R +rw .
+
+            function unescapeNupkgPaths () {
```

Comment:
Can we leave this for a separate PR?  It shouldn't be needed for `wiseunpacker` anymore, and it will require a lot of packages to be tested.

Reply:
Sounds good, I'll move it to a new PR

Reply:
https://github.com/NixOS/nixpkgs/pull/337096